### PR TITLE
MOSIP-32461, MOSIP-36412 Changing commons  versions for 1.3.x branch

### PR DESCRIPTION
**Pull request by Khuddusshariff0022**

### CODE REVIEW

File: registration-processor/pom.xml [hunk 1]
```diff
@@ -127,24 +127,24 @@
 		<registration.processor.message.sender.version>1.3.0-SNAPSHOT</registration.processor.message.sender.version>
 
 		<!-- Kernel -->
-		<kernel.core.version>1.3.0-SNAPSHOT</kernel.core.version>
-		<kernel.dataaccess.hibernate.version>1.3.0-SNAPSHOT</kernel.dataaccess.hibernate.version>
+		<kernel.core.version>1.3.0-beta.1</kernel.core.version>
+		<kernel.dataaccess.hibernate.version>1.3.0-beta.1</kernel.dataaccess.hibernate.version>
 		<kernel.templatemanager.velocity.version>1.3.0-SNAPSHOT</kernel.templatemanager.velocity.version>
```
Comment (ashok-ksharma) [inline registration-processor/pom.xml]: We should change <kernel.templatemanager.velocity.version> this to 1.3.0-beta.1

File: registration-processor/pom.xml [hunk 2]
```diff
@@ -127,24 +127,24 @@
...
 		<object.store.khazana.version>1.3.0-SNAPSHOT</object.store.khazana.version>
 		<kernel.biometrics.api.version>1.3.0-SNAPSHOT</kernel.biometrics.api.version>
-		<kernel-keymanager-service.version>1.3.0-SNAPSHOT</kernel-keymanager-service.version>
-		<kernel.websubclient.version>1.3.0-SNAPSHOT</kernel.websubclient.version>
+		<kernel-keymanager-service.version>1.3.0-beta.1</kernel-keymanager-service.version>
+		<kernel.websubclient.version>1.3.0-beta.1</kernel.websubclient.version>
 		<kernel-biosdk-provider.version>1.3.0-SNAPSHOT</kernel-biosdk-provider.version>
-		<kernel.bom.version>1.3.0-SNAPSHOT</kernel.bom.version>
+		<kernel.bom.version>1.3.0-beta.1</kernel.bom.version>
 		<sonar.coverage.exclusions>**/dto/**,
```
Comment (ashok-ksharma) [inline registration-processor/pom.xml:149]: 1. Do we need this bom dependency here or we can change the bom version with property: https://github.com/mosip/registration/blob/release-1.3.x/registration-processor/core-processor/registration-processor-verification-stage/pom.xml#L33
2. Do we need to define this or we can directly refer from parent pom : https://github.com/mosip/registration/blob/release-1.3.x/registration-processor/registration-processor-notification-service/pom.xml#L19

**Review by ashok-ksharma — CHANGES_REQUESTED** [review]

File: registration-processor/pom.xml (hunk 1 above)
Comment (Khuddusshariff0022) [reply registration-processor/pom.xml]: done.

File: registration-processor/pom.xml (hunk 2 above)
Comment (Khuddusshariff0022) [reply registration-processor/pom.xml:149]: 1. For bom dependency made it to refer property for version.

2. shifted the property to parent pom.

**Review by ashok-ksharma — APPROVED** [review]

**Review by ckm007 — APPROVED** [review]

**Pull request merged**